### ISSUE TITLE
General question: model fitting

### ISSUE THREAD

**Issue by ArjunZ786** [Rubikplayer/flame-fitting#41]
Does it make sense to fit the scan data to the model twice? 
Like first time keeping shape weights more regularized in hopes of getting a better initial alignment and then when the alignment looks right make it more de-regularized to get a tighter fit.

**Comment by TimoBolkart** [comment]
Yes, that makes perfectly sense. We commonly follow a similar optimization strategy.
